test/system/src/main/java/io/pravega/test/system/framework/services/docker/HDFSDockerService.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.system.framework.services.docker;
+
+import com.spotify.docker.client.exceptions.DockerException;
+import com.spotify.docker.client.messages.ContainerConfig;
+import com.spotify.docker.client.messages.ServiceCreateResponse;
+import com.spotify.docker.client.messages.mount.Mount;
+import com.spotify.docker.client.messages.swarm.ContainerSpec;
+import com.spotify.docker.client.messages.swarm.EndpointSpec;
+import com.spotify.docker.client.messages.swarm.NetworkAttachmentConfig;
+import com.spotify.docker.client.messages.swarm.PortConfig;
+import com.spotify.docker.client.messages.swarm.ResourceRequirements;
+import com.spotify.docker.client.messages.swarm.Resources;
+import com.spotify.docker.client.messages.swarm.ServiceMode;
+import com.spotify.docker.client.messages.swarm.ServiceSpec;
+import com.spotify.docker.client.messages.swarm.TaskSpec;
+import lombok.extern.slf4j.Slf4j;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+
+@Slf4j
+public class HDFSDockerService extends DockerBasedService {
+
+    private int instances = 1;
+    private double cpu = 0.1;
+    private double mem = 2048.0;
+
+    public HDFSDockerService(final String serviceName) {
+        super(serviceName);
+    }
+
+    @Override
+    public void stop() {
+        try {
+            dockerClient.removeService(getID());
+        } catch (DockerException | InterruptedException e) {
+            log.error("unable to remove service {}", e);
+        }
+    }
+
+    @Override
+    public void clean() {
+    }
+
+    @Override
+    public void start(final boolean wait) {
+        try {
+            ServiceCreateResponse serviceCreateResponse = dockerClient.createService(setServiceSpec());
+            if (wait) {
+                waitUntilServiceRunning().get(5, TimeUnit.MINUTES);
+            }
+            assertThat(serviceCreateResponse.id(), is(notNullValue()));
+        } catch (InterruptedException | DockerException | TimeoutException | ExecutionException e) {
```

Comment:
Please don't suppress exception like this.

Reply:
done.